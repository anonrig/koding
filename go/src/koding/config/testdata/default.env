KONFIG_KLOUD_KEYGENBUCKET=undefined
KONFIG_KLOUD_REGION=default
KONFIG_KLOUD_REGISTERURL=http://94.254.191.51:8090/kloud/kite
KONFIG_KLOUD_KONTROLURL=http://94.254.191.51:8090/kontrol/kite
KONFIG_KLOUD_TUNNELURL=http://dev-t.koding.com