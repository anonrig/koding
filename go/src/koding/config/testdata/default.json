{
  "goGenerate": {
    "buckets": {
      "publicLogs": {
        "name": "kodingdev-publiclogs",
        "region": "us-east-1"
      }
    },
    "endpoints": {
      "ip": "https://dev-p2.koding.com/-/ip",
      "ipCheck": "https://dev-p2.koding.com/-/ipcheck",
      "kdLatest": "https://koding-kd.s3.amazonaws.com/development/latest-version.txt",
      "klientLatest": "https://koding-klient.s3.amazonaws.com/{{.Environment}}/latest-version.txt",
      "kloud": "https://sandbox.koding.com/kloud/kite",
      "kontrol": "https://sandbox.koding.com/kontrol/kite",
      "tunnelServer": "http://dev-t.koding.com/kite"
    },
    "environment": "default",
    "routes": {
      "dev.koding.com": "127.0.0.1"
    }
  }
}
