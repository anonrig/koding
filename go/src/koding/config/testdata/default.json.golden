{
  "buckets": {
    "publiclogs": [
      {
        "environment": [
          "default"
        ],
        "name": "undefined",
        "region": "default"
      }
    ]
  },
  "endpoints": {
    "ip": [
      {
        "environment": [
          "production",
          "managed"
        ],
        "url": "https://p.koding.com/-/ip"
      },
      {
        "environment": [
          "default",
          "dev",
          "development",
          "sandbox",
          "devmanaged"
        ],
        "url": "https://dev-p2.koding.com/-/ip"
      }
    ],
    "ipcheck": [
      {
        "environment": [
          "production",
          "managed"
        ],
        "url": "https://p.koding.com/-/ipcheck"
      },
      {
        "environment": [
          "default",
          "dev",
          "development",
          "sandbox",
          "devmanaged"
        ],
        "url": "https://dev-p2.koding.com/-/ipcheck"
      }
    ],
    "kd-latest": [
      {
        "environment": [
          "production",
          "managed"
        ],
        "url": "https://koding-kd.s3.amazonaws.com/production/latest-version.txt"
      },
      {
        "environment": [
          "default",
          "dev",
          "development",
          "sandbox",
          "devmanaged"
        ],
        "url": "https://koding-kd.s3.amazonaws.com/development/latest-version.txt"
      }
    ],
    "klient-latest": [
      {
        "environment": [
          "default",
          "dev",
          "development",
          "sandbox"
        ],
        "url": "https://koding-klient.s3.amazonaws.com/development/latest-version.txt"
      },
      {
        "environment": [
          "production"
        ],
        "url": "https://koding-klient.s3.amazonaws.com/production/latest-version.txt"
      },
      {
        "environment": [
          "managed"
        ],
        "url": "https://koding-klient.s3.amazonaws.com/managed/latest-version.txt"
      },
      {
        "environment": [
          "devmanaged"
        ],
        "url": "https://koding-klient.s3.amazonaws.com/devmanaged/latest-version.txt"
      }
    ],
    "kloud": [
      {
        "environment": [
          "default"
        ],
        "url": "http://94.254.191.51:8090/kloud/kite"
      }
    ],
    "kontrol": [
      {
        "environment": [
          "default"
        ],
        "url": "http://94.254.191.51:8090/kontrol/kite"
      }
    ],
    "tunnelserver": [
      {
        "environment": [
          "default"
        ],
        "url": "http:/dev-t.koding.com/kite"
      }
    ]
  },
  "routes": {
    "dev.koding.com": "127.0.0.1"
  }
}